gm4_potion_swords/data/gm4/advancements/potion_swords.json

```diff
@@ -2,7 +2,7 @@
   "display": {
     "icon": {
       "item": "golden_sword",
-      "nbt": "{display:{Name:'\"potion_swords logo\"'}}"
+      "nbt": "{CustomModelData:1,display:{Name:'\"potion_swords logo\"'}}"
```

Comment:
We can remove the display name on this item. That was a pre-CMD hack that would allow optifine to retexture the advancements based upon names.